src/parser/core/modules/CooldownDowntime.js

```diff
@@ -22,6 +22,7 @@ export default class CooldownDowntime extends Module {
 		this.allowedDowntime = 0
 		//Default Target to hit
 		this.target = 95
+		this.description = <Trans id="core.cooldowndowntime.ogcd-cd-metric">Always make sure to use your OGCDs when they are up but don't clip them.  {this.allowedDowntime === 0 ? '' : <Trans id="core.cooldowndowntime.ogcd-cd-buffer">To account for random factors you are given a buffer of {this.parser.formatDuration(this.allowedDowntime/1000)} seconds per instance to hold your cooldowns.</Trans>}</Trans>
```

Comment:
Sorry, I should've clarified. `this.parser.formatDuration()` takes milliseconds and includes "s" to denote seconds (e.g. `this.parser.formatDuration(10000)` produces `'10s'`), then uses `m:ss` notation if it exceeds a minute. You'd want to turn this:
```js
{this.parser.formatDuration(this.allowedDowntime/1000)} seconds
```
into this:
```js
{this.parser.formatDuration(this.allowedDowntime)}
```
to utilize it properly.

Reply:
ah ok. I thought it took the value in seconds. Didn't doublecheck :blobsweat: